src/pretix/base/services/update_check.py

```diff
@@ -0,0 +1,125 @@
+import sys
+import uuid
+from datetime import timedelta
+
+import requests
+from django.dispatch import receiver
+from django.utils.timezone import now
+from django.utils.translation import ugettext_lazy as _, ugettext_noop
+from i18nfield.strings import LazyI18nString
+
+from pretix import __version__
+from pretix.base.models import Event
+from pretix.base.plugins import get_all_plugins
+from pretix.base.services.mail import mail
+from pretix.base.settings import GlobalSettingsObject
+from pretix.base.signals import periodic_task
+from pretix.celery_app import app
+from pretix.helpers.urls import build_absolute_uri
+
+
+@receiver(signal=periodic_task)
+def run_update_check(sender, **kwargs):
+    gs = GlobalSettingsObject()
+    if not gs.settings.update_check_perform:
+        return
+
+    if not gs.settings.update_check_last or now() - gs.settings.update_check_last > timedelta(hours=23):
+        update_check.apply_async()
+
+
+@app.task
+def update_check():
+    gs = GlobalSettingsObject()
+    if not gs.settings.update_check_perform:
+        return
+
+    if not gs.settings.update_check_id:
+        gs.settings.set('update_check_id', uuid.uuid4().hex)
+
+    if 'runserver' in sys.argv:
+        gs.settings.set('update_check_last', now())
+        gs.settings.set('update_check_result', {
+            'error': 'development'
+        })
+        return
+
+    check_payload = {
+        'id': gs.settings.get('update_check_id'),
+        'version': __version__,
+        'events': {
+            'total': Event.objects.count(),
+            'live': Event.objects.filter(live=True).count(),
+        },
+        'plugins': [
+            {
+                'name': p.module,
+                'version': p.version
+            } for p in get_all_plugins()
```

Comment:
You might want to send `gs.settings['update_check_result']` if it contains `error`, here, so you know that prior updates failed.